exercises/pov/stack.yaml

```diff
@@ -1 +1,6 @@
 resolver: lts-8.2
+
+packages:
+  - location: .
+  - location: ./common/test-prelude
```

Comment:
I was thinking about it, and maybe using a package outside the exercise's folder could be problematic. Two exercises fetched at different times could need distinct versions of the library.

I seems that this would only work would be with versioned packages.